---
VLP16_DiagnosticsRawMessage  msg
